scripts/docker/eth.dockerfile

```diff
@@ -0,0 +1,23 @@
+# Mutistage Dockerfile for the "eth" Ethereum node.
+
+# Build stage
+FROM alpine:latest as builder
+RUN apk add --no-cache \
+        git \
+        cmake \
+        g++ \
+        make \
+        linux-headers \
+        leveldb-dev --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing/
+RUN git clone --recursive https://github.com/ethereum/cpp-ethereum --branch develop --single-branch --depth 1
```

Comment:
I'm also not an expert, but I think if you use multiple run-statements, the intermediate snapshots will be stored, so your image will be larger.